tpcx_bb/queries/q08/tpcx_bb_query_08_sql.py

```diff
@@ -199,10 +199,10 @@ def main(data_dir, client, bc, config):
         INNER JOIN web_page_2 ON wcs_web_page_sk = wp_web_page_sk
         WHERE wcs_user_sk IS NOT NULL
         AND wcs_click_date_sk BETWEEN {q08_start_dt} AND {q08_end_dt}
+        ORDER BY wcs_user_sk
```

Comment:
Can you add a comment indicating why this is here? I can imagine a future scenario in which we want to remove this step.

Reply:
ORDER BY no ensure `wcs_user_sk` values are in the same partition. Then just for now adding the ORDER BY to avoid the iisue above.

Reply:
Tested using the **dgx201** at sf **1K** 

```
(bsql-tpcx-bb) christianc@rl-dgx2-d17-u05-rapids-dgx201:~/tpcx-bb/tpcx_bb/queries/q08$ python tpcx_bb_query_08_sql.py --config_file=/home/christianc/yaml_dir/validation_1k_config.yaml
Using default arguments
Connected to tcp://10.150.162.27:8786
BlazingContext ready
/home/christianc/miniconda3/envs/bsql-tpcx-bb/lib/python3.7/site-packages/dask_cudf/core.py:329: UserWarning: Rearranging data by column hash. Divisions will lost. Set ignore_index=False to preserve Index values.
  "Rearranging data by column hash. Divisions will lost. "
Standard ETL Query
/home/christianc/miniconda3/envs/bsql-tpcx-bb/lib/python3.7/site-packages/pandas/util/__init__.py:12: FutureWarning: pandas.util.testing is deprecated. Use the functions in the public API at pandas.testing instead.
  import pandas.util.testing
Correctness Assertion True
```

Reply:
Thanks. We'll need to keep the hash repartition call then to guarantee correctness, which I see you've added back 👍